### ISSUE TITLE
Invert sizing seems to be broken for ingest

### ISSUE THREAD

**Issue by LizziOh** [aodn/nrmn-application#1343]
https://nrmn.aodn.org.au/data/job/498/edit

I'm trying to ingest data from this file and the validation mechanism is throwing warnings for 700cm Jasus edwardsii weven though Invert sizing is valid for the ingest job and in the IsInvertSized column.

![image](https://github.com/aodn/nrmn-application/assets/52521522/c8389495-ee4d-4ad2-9348-0cc5dca04e77)

I have recently ingested ATRC data and did not notice this issue

I very much need this rectified before ingest.  The problem seems to be the validations when outside the l5/L95 are reading the non-invert sizes (fish sizes).  But we need to know that on ingest these are not recorded under the fish sizes (ie. is it just the display of the validation warning OR an ingest problem). Is it possible to confirm this ASAP as it another thing holding up data ingest?

@utas-raymondng @bpasquer @atcooper1

**Comment by bpasquer** [comment]
@LizziOh 
I've reproduced you issue and after some investigation these are my findings:
- all errors are correctly reported (ie, they are outside the L5/95 ) but the message translates the indicated size into Fish units _**in the majority of cases**_ . Unfortunately- (or fortunately)  errors are also reported in the correct unit (see the screenshot with the 20cm Jasus). 
![Screenshot from 2024-03-15 16-21-21](https://github.com/aodn/nrmn-application/assets/3733612/39146718-8dfa-48db-a463-dfe179692a7b)
- **data are recorded  in the database in the correct size units** 
 
In conclusion the issue seems to be with the unit mapping in the validation message.
However, the inconsistency in behavior suggest there are other things at play here , but  I have been unable to fin any clear patterns yet.
But importantly, the data is recorded in the correct size units.
